src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Common.Test/FilePathNormalizerTest.cs

```diff
@@ -42,27 +42,27 @@ public void NormalizeDirectory_EndsWithSlash()
         {
             // Arrange
             var filePathNormalizer = new FilePathNormalizer();
-            var directory = "\\path\\to\\directory\\";
+            var directory = "C:\\path\\to\\directory\\";
```

Comment:
Am I looking at this wrong or would this fail if we ran the test on Linux?

Reply:
You'd think but even on Linux the runtime respects these types of paths

Reply:
That is shocking. I am shocked.